ism330dhcx_reg.h

```diff
@@ -2836,12 +2836,12 @@ typedef union
  * The __weak directive allows the final application to overwrite
  * them with a custom implementation.
  */
-int32_t __weak ism330dhcx_read_reg(stmdev_ctx_t *ctx, uint8_t reg,
-                                   uint8_t *data,
-                                   uint16_t len);
-int32_t __weak ism330dhcx_write_reg(stmdev_ctx_t *ctx, uint8_t reg,
-                                    uint8_t *data,
-                                    uint16_t len);
+int32_t ism330dhcx_read_reg(stmdev_ctx_t *ctx, uint8_t reg,
+                            uint8_t *data,
+                            uint16_t len);
+int32_t ism330dhcx_write_reg(stmdev_ctx_t *ctx, uint8_t reg,
+                             uint8_t *data,
+                             uint16_t len);
 

```

Comment:
Ciao Carlo,

Can you possibly put this change in a separate commit? I'd prefer to have this change separated from the rest of warnings. Thanks! (rest is absolutely fine!)

Reply:
Ok, now it should be fine